net-fs/samba/samba-4.15.3.ebuild

```diff
@@ -212,6 +212,7 @@ multilib_src_configure() {
 		$(multilib_native_use_with ads)
 		$(multilib_native_use_enable ceph cephfs)
 		$(multilib_native_use_with cluster cluster-support)
+		$(usex cpu_flags_x86_aes "--accel-aes=intelaesni" "--accel-aes=none")
```

Comment:
Wonder if --accel-aces=$(usex cpu_flags_aes intelaesni none) would be nicer here, maybe.

Also, I'd use Closes: instead in the commit message.

Reply:
Where in myconf should that line go, to keep things tidy?

Reply:
No preference from me.